tests/gis_tests.cc

```diff
@@ -0,0 +1,84 @@
+#include <gtest/gtest.h>
+
+#include <iostream>
+#include <string>
+#include "gis.h"
+
+using namespace std;
+
+using cyclus::GIS;
```

Comment:
Good tests. We need to determine a way to test that this class can be mixed into an agent object.